### DISCUSSION TITLE
How do I give players permissions

### DISCUSSION THREAD

**Discussion by ThPhoenixGamer225** [ChanceSD/PvPManager#461] [Q&A] [answered]
I can see there are three different tiers of permissions, but I can't find anywhere in the Ini files to give a tier of permissions to anyone. I am trying to have it where players can turn their pvp status on or off, but it seems only to allow admins to change their own status. Am I just missing something?

**Comment by ChanceSD** [comment, marked as answer]
> I can see there are three different tiers of permissions, but I can't find anywhere in the Ini files to give a tier of permissions to anyone. I am trying to have it where players can turn their pvp status on or off, but it seems only to allow admins to change their own status. Am I just missing something?

To allow players to change their own status, give them the permission `pvpmanager.pvpstatus.change`, this is done with your permissions plugin. Most people use LuckPerms currently.
